Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -28,7 +28,7 @@ extension KeyboardShortcuts {
 	public final class RecorderCocoa: NSSearchField, NSSearchFieldDelegate {
 		private let minimumWidth: Double = 130
 		private var eventMonitor: LocalEventMonitor?
-		private let shortcutName: Name
+		private var shortcutName: Name
```

Comment:
Instead of a `updateShortCutName` method, you can just make this property public and use `didSet`.